tests/test_device.py

```diff
@@ -340,6 +340,8 @@ def test_check_validity_on_valid_queue(self, mock_device_supporting_paulis):
         # Raises an error if queue or observables are invalid
         dev.check_validity(queue, observables)
 
+    # Works with opmath disabled by replacing
+    # match="Observable Prod not supported on device MockDevice"   with   match="Tensor observables not supported"
     def test_check_validity_on_tensor_support(self, mock_device_supporting_paulis):
```

Comment:
The test case is called `test_check_validity_on_tensor_support` but then it checks `Prod` support? What was the original purpose of this test case? I'm not sure if changing the expected error message makes much sense here?

Reply:
This is already updated in another PR

Reply:
Edit: let's ignore these two tests in this PR